rubin_scheduler/scheduler/basis_functions/basis_functions.py

```diff
@@ -970,6 +971,47 @@ def _calc_value(self, conditions, indx=None):
         return result
 
 
+class SlewtimeNoBandBasisFunction(BaseBasisFunction):
```

Comment:
Why not just have `SlewtimeBasisFunction` take `None` as an option on the `bandname`?

Reply:
I was going to do that .. I just figured this was enough of a change that it would be worthwhile to be REALLY clear that it was different than the basic SlewtimeBasisFunction. 
If you think it would be ok, I'm happy to change it.

Reply:
I think the standard on most of the `Features` is to have a band set, and then if it is `None` it takes any band. So might as well have the same convention here.

Reply:
Done.